src/utils/format_stackdriver/format_stackdriver.c

```diff
@@ -134,16 +134,23 @@ static int format_typed_value(yajl_gen gen, metric_t const *m,
 
   switch (m->family->type) {
   case METRIC_TYPE_GAUGE: {
-    int status = json_string(gen, "doubleValue");
-    if (status != 0)
+    int status =
+        json_string(gen, "doubleValue") || yajl_gen_double(gen, m->value.gauge);
```

Comment:
Because `||` operation results in `status` value being always a `bool`, I think in all instances of this pattern, it could actually declared as `bool` to help compiler better catch mistakes:
```suggestion
    bool status =
        json_string(gen, "doubleValue") || yajl_gen_double(gen, m->value.gauge);
```

Other alternative would be using something like the `ERR_COMBINE()` macro to preserve also other return values than 0/1.

This is large enough change that it should be in another PR though.